### DISCUSSION TITLE
Having AITool Images show in BI Alerts List

### DISCUSSION THREAD

**Discussion by johnseybert** [VorlonCD/bi-aidetection#258] [Q&A] [answered]
I have been lurking and learning with AITool since back when GentlePumpkin first introduced it.  Of course it has advanced lightyears at this point, and I am still trying to figure much of it out.  A big thank you to VorlonCD and all have have worked so hard to develop this program.

I am not a programmer, but stumble my way around pretty well. I am trying to get the images generated by AITool when it detects someone (with the detection box and label) to show up in my BI "Alerts" list.  I am able to send a copy of the image to the BI Alerts folder, but it does not show in the BI list.  I am guessing this is because BI does not have any reference to it in its database.  I am curious if anyone has been able to do this or has some idea how to get it to show.

I have tried the BI/Deepstack integration, but it is just too slow for me for some reason.  For now I am sticking to AITool since it seems to perform better and has better detection.

I appreciate any help that could point me in the right direction if this is possible or not.  Thanks again to all who continue the development!

**Comment by VorlonCD** [comment, marked as answer]
Doesnt the default trigger URL work?   The one you get if you delete the URL, uncheck and recheck.  Or create a new camera?

[BlueIrisURL]/admin?trigger&camera=[camera]&user=[Username]&pw=[Password]&flagalert=1&memo=[summary]&jpeg=[ImagePathEscaped]

Its flagalert=1 and Jpeg= that should do the trick along with telling BI to only show flagged alerts.

**Reply by johnseybert** [reply]
Ah Ha! I was missing the "&jpeg=[ImagePathEscaped]".  Maybe since I have been using the same trigger URL from version to version, it never got updated to include the image path at some point? The default definitely solved the problem.  Thank you!
